docs/developers/cosmwasm-dependency.md

```diff
@@ -7,15 +7,14 @@ sidebar_label: CosmWasm Dependencies
 ## Environment Setup
 
 For this tutorial, we will be using `curl` and `jq` as helpful
-tools.
-
-You can follow the guide on installing them [here](../nodes/environment.mdx#setting-up-dependencies).
+tools. You can follow the guide on installing them
+[here](../nodes/environment.mdx#setting-up-dependencies).
 
 ## Golang Dependency
 
-The Golang version used for this tutorial is v1.18+
+The Golang version used for this tutorial is v1.18.2+.
```

Comment:
I would keep it as v1.18+ which implies v1.18.2 is supported

Reply:
done! thank you